anda/tools/arduino-app-lab-bin/arduino-app-lab-bin.spec

```
@@ -0,0 +1,56 @@
+%global appid cc.arduino.AppLab
+
+Name:           arduino-app-lab-bin
+Version:        0.2.4
+Release:        1%{?dist}
+Summary:        A powerful visual environment for managing the Arduino UNO Q
+
+URL:            https://www.arduino.cc/en/software
+License:        GPL-3.0
+
+Source0:        https://downloads.arduino.cc/AppLab/Stable/ArduinoAppLab_%{version}_Linux_x86-64.tar.gz
+Source1:        https://downloads.arduino.cc/AppLab/Stable/source-app-lab-%{version}.zip
+Source2:        cc.arduino.AppLab.desktop
+Source3:        cc.arduino.AppLab.metainfo.xml
+
+ExclusiveArch:  x86_64
+
+Requires:       android-tools
+
+BuildRequires:  terra-appstream-helper
+
+Suggests:       arduino-flasher-cli arduino-app-cli  
+
+Packager:       Jaiden Riordan <jade@fyralabs.com>
+
+%description
+%summary.
+
+%prep
+tar -xvf %{_sourcedir}/ArduinoAppLab_%{version}_Linux_x86-64.tar.gz
+unzip %{_sourcedir}/source-app-lab-%{version}.zip
+
+%install
+install -dm755 %{buildroot}%{_bindir}
+install -p -m755 ArduinoAppLab_%{version}_Linux_x86-64/arduino-app-lab %{buildroot}%{_bindir}/%{name}
+
+install -dm755 %{buildroot}%{_iconsdir}/hicolor/scalable/apps/
+install -p -m644 source-app-lab/ui-packages/images/assets/round-arduino-logo.svg %{buildroot}%{_iconsdir}/hicolor/scalable/apps/cc.arduino.AppLab.svg
+
+install -dm755 %{buildroot}%{_datadir}/applications/
+install -p -m644 %{SOURCE2} %{buildroot}%{_datadir}/applications/%{appid}.desktop
+
+install -p -m644 source-app-lab/LICENSE %{buildroot}%{_defaultlicensedir}/%{name}/LICENSE
+
```

This shit is a hack but makes it 8 million times easier.
```suggestion
cp source-app-lab/LICENSE -t .
cp source-app-lab/dependency_licenses -t .
```

---

why not use install?